ydb/docs/en/core/yql/reference/yql-core/syntax/create_view.md

```diff
@@ -0,0 +1,88 @@
+# CREATE VIEW
+
+CREATE VIEW defines a view of a query.
+
+A view is a logical representation of a table formed by the specified query. The view does not physically store the table, but runs the query to produce the data whenever the view is selected from.
+
+## Syntax
+
+```sql
+CREATE VIEW name
+[ WITH ( view_option_name [= view_option_value] [, ... ] ) ]
+AS query
+```
+
+### Parameters
+
+* `name`
+
+The name of the view to be created. The name must be distinct from the name of any other schema object.
+* `query`
+
+The SELECT query, which will be used to produce the logical table the view represents.
+
+`WITH ( view_option_name [= view_option_value] [, ... ] )`
+
+This clause specifies optional parameters for a view. The following parameters are supported:
+
+* `security_invoker` (Bool) {#security_invoker}
+
+This option causes the underlying base relations to be checked against the privileges of the user of the view rather than the view owner.
```

Comment:
```suggestion
* `security_invoker` (Bool) {#security_invoker} - this option causes the underlying base relations to be checked against the privileges of the user of the view rather than the view owner.
```